### PR TITLE
Prepare release 0.3.3

### PR DESCRIPTION
**Pull request by PaulJonasJost**

### CODE REVIEW

**Review by Doresic — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,6 +6,23 @@ Release notes
 ..........
 
 
+0.3.2 (2023-10-02)
+-------------------
+
+* Visualize:
+    * Get optimization result by id (#1116)
+* Storage:
+    * {id} in history storage filename (#1118)
```
Comment (dilpath) [inline CHANGELOG.rst:15]: {id} is not so descriptive to me, but fine as is

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,6 +6,23 @@ Release notes
...
+* Objective:
+    * adjusted PEtab.jl syntax to new release(#1128, #1131)
+    * Documentation on PEtab importer (#1126)
+* Ensembles
+    * Additional option for cutoff calculation (#1124)
+    * fix ensembles from optimization endpoints (#1130)
```
Comment (dilpath) [inline CHANGELOG.rst:21]: Also not so descriptive

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,6 +6,23 @@ Release notes
...
 
+0.3.2 (2023-10-02)
+-------------------
+
+* Visualize:
+    * Get optimization result by id (#1116)
+* Storage:
+    * {id} in history storage filename (#1118)
+* Objective:
+    * adjusted PEtab.jl syntax to new release(#1128, #1131)
```
Comment (dilpath) [inline CHANGELOG.rst:17]:
```suggestion
    * adjusted PEtab.jl syntax to new release (#1128, #1131)
```

**Review by dilpath — APPROVED** [review]
:+1:

**Pull request merged**